apps/aecore/src/aec_keys.erl

```diff
@@ -239,12 +239,12 @@ handle_call({sign, Tx}, _From,
             SignedTx = aetx_sign:sign(Tx, PrivKey, CryptoMap),
             {reply, {ok, SignedTx}, State}
     end;
-handle_call({verify, Sigs, Tx}, _From, #state{crypto = C} = State) ->
-    Signers = aetx:signers(Tx),
+handle_call({verify, Signatures, Tx}, _From, #state{crypto = C} = State) ->
```

Comment:
Ok.

We will also need an API function that checks e.g. wether the `initiator` alone has signed. This would be used in the off-chain logic, where e.g. the `initiator` signs the `channel_create_tx`, then passes it to the `participant`. The `participant` should verify that the `initiator` signature is in fact present and valid, before adding its own signature.

Reply:
OK, So this should have different arity then? Take additional param, which is account's pubkey that should have signed?